### PR TITLE
Structure change(examples rename-in) and readme's added for each of examples

### PR DESCRIPTION
**Pull request by krzysztofkrzeslak**

### CODE REVIEW

File: 1_send_data/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Send data to LiveObjects
```
Comment (mdelain) [inline 1_send_data/README.md:1]: Don't forget the space in "Live Objects" :)

File: 1_send_data/arduino_secrets.h [hunk 2]
```diff
@@ -9,4 +9,4 @@ const char SECRET_APN_PASS[] = "";		// specify the password for your APN (if nee
 	// You API key must have at least the predefined 'MQTT Device' rights profile
 	// (alternatively: 'Device Access' read + write rights if need to customise the rights).
 	// Please note that you *must* use a TLS connection (MQTTS) if you grant more rights to the API key.
-const char SECRET_LIVEOBJECTS_API_KEY[] = "...";
+const char SECRET_LIVEOBJECTS_API_KEY[] = "bf4ef02c497345128ecda296f15e9f02";
```
Comment (mdelain) [inline 1_send_data/arduino_secrets.h]: Tell me it is not a mistake and this is not your actual API key ;)
We should be consistent on credential example, either we put a sample value or we just leave them empty - but I couldn't decide for the API key and left "...". Moreover, I put a [verification routine in the .h file](https://github.com/mdelain/Arduino_MKR_NB_1500/blob/master/Simple_parameter/LiveObjects.h#L487) (code stops if API key != 32).
Arduino examples usually leave them empty.

File: 2_simple_parameters/README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
-# Send data to LiveObjects
+# LiveObjects commands
```
Comment (mdelain) [inline 2_simple_parameters/README.md:1]: Should be "parameters"?

File: 3_Parameter_with_callback/README.md [hunk 4]
```diff
@@ -1,22 +1,22 @@
-# LiveObjects commands
+# LiveObjects parameters with callbacks
 
 Parameters give you ability to configure device device from LiveObjects portal, in this example we will use parameter to adjust time rate on which messages from device are sent.<br>
 
-![diagram](img/command_diagram.png)
+![diagram](img/parameter_diagram.png)
 
 Additionally in this example we will use "callback" function which will convert value(in seconds) received in command to arduino native delay milliseconds:
```
Comment (mdelain) [inline 3_Parameter_with_callback/README.md:7]: missing space before '(in seconds)'

File: 3_Parameter_with_callback/README.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ If all went fine under **devices** tab on Live LiveObjects portal you should see
 **Is device sending data:**<br>
 Under data tab on LiveObjects portal you should see messages from stream identified by modem IMEI, along with values *{ "uptime": xxxxx }*
 
-## Send command
+## Send parameter
```
Comment (mdelain) [inline 3_Parameter_with_callback/README.md:29]: better 'Update parameter'

File: 1_send_data/README.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# Send data to LiveObjects
+
+This example show how to send some example data(device uptime) to LiveObjects using Arduino MKR1500
```
Comment (mdelain) [inline 1_send_data/README.md:3]: 'This example shows how to send some sample data (device uptime) to Live Objects using Arduino MKR NB 1500.'

File: 2_simple_parameters/README.md (hunk 3 above)
Comment (mdelain) [inline 2_simple_parameters/README.md:1]: should be 'parameters'

File: 1_send_data/arduino_secrets.h (hunk 2 above)
Comment (krzysztofkrzeslak) [reply 1_send_data/arduino_secrets.h]: Of course, it was just "example" key ;)

File: 3_Parameter_with_callback/README.md (hunk 5 above)
Comment (krzysztofkrzeslak) [reply 3_Parameter_with_callback/README.md:29]: Agree! Will fix in some of next pull requests as I see you already merged it.

File: 2_simple_parameters/README.md (hunk 3 above)
Comment (krzysztofkrzeslak) [reply 2_simple_parameters/README.md:1]: Also good point, thanks for watchfulness!

**Comment by krzysztofkrzeslak** [comment]
Hi, good points, thanks for watchfulness I see you already fixed some of flaws, thanks! anyway I will go through them tomorrow one more time for a double-check.

**Pull request merged**